docs/template_extension/old/fft_kernel.cpp

```diff
@@ -82,8 +82,10 @@ InferenceEngine::StatusCode FFTImpl::init(InferenceEngine::LayerConfig& config,
 static cv::Mat infEngineBlobToMat(const InferenceEngine::Blob::Ptr& blob) {
     // NOTE: Inference Engine sizes are reversed.
     std::vector<size_t> dims = blob->getTensorDesc().getDims();
-    std::vector<int> size(dims.begin(), dims.end());
-    auto precision = blob->getTensorDesc().getPrecision();
+    std::vector<int> size(dims.size());
+    for (size_t i = 0; i < dims.size(); i++)
+        size[i] = static_cast<int>(dims[i]);
```

Comment:
we could use std::transform

Reply:
Yes, I will change all such places before the merge.
Now first time goal is to fix CI